src/test/java/edu/ucsb/mapache/advice/AuthControllerAdviceTests.java

```diff
@@ -74,6 +81,15 @@ public void test_getUser_createNewUserAndAdmin() {
     verify(mockAdminRepository, times(1)).save(new Admin(exampleUser.getEmail(), true));
   }
 
+  @Test
+  public void test_getUser_createNewUserAndSlackAdmin() {
+    when(mockAppUserRepository.save(any(AppUser.class))).thenReturn(exampleUser);
+    when(mockSlackUserRepository.findAdminByEmail(exampleUser.getEmail()))
+      .thenReturn(new ArrayList<SlackUser>(Arrays.asList(new SlackUser("fakeID", "Fake Name", "Fake Name", new SlackUserProfile(exampleUser.getEmail())))));
```

Comment:
Are these 6 consecutive closing parentheses okay? Looks kind of gross, but I couldn't think of a way to make it look nicer.

Reply:
A nicer way to do it could be to create temp SlackUser and SlackUserProfile objects before the line and then feed them into the function. Thought I don't think it matters that there are lots of parentheses.